www/content/installation/index.md

```diff
@@ -74,12 +73,11 @@ D’autres méthodes d’installation sont possibles, en passant le [fichier
 source][] à [XKalamine][].
 
 Remarque : avec certains bureaux (Gnome notamment), la touche morte [★]{.odk}
-ne fonctionne que si Ergo‑L est défini comme disposition par défaut, i.e. en
-haut de la liste dans les préférences clavier.
```

Comment:
mon latinisme !!!

(line-wrap STP)

Reply:
En français, « c’est-à-dire » ou « c.-à-d. » (pas très beau, ni très économique, *but I don’t make the rules*), n’est-ce pas ?

Reply:
tu peux l’écrire en toutes lettres tant que la ligne est tronquée à 80 colonnes :-)